components/encryption/src/manager/mod.rs

```diff
@@ -100,14 +103,24 @@ impl Dicts {
         }
         let key_bytes = self.key_dict.write_to_bytes()?;
         file.write(&key_bytes, master_key)?;
+
+        ENCRYPTION_FILE_SIZE_GAUGE
+            .with_label_values(&["key_dictionary"])
+            .set(key_bytes.len() as _);
+        ENCRYPTION_INITIALIZED_GAUGE.set(1);
+        ENCRYPTION_DATA_KEY_NUM_GAUGE.set(self.key_dict.keys.len() as _);
         Ok(())
     }
 
     fn save_file_dict(&self) -> Result<()> {
         let file = EncryptedFile::new(&self.base, FILE_DICT_NAME);
         let file_bytes = self.file_dict.write_to_bytes()?;
         // File dict is saved in plaintext.
-        file.write(&file_bytes, &PlaintextBackend::default())?;
+        file.write(&file_bytes, &PlainTextBackend::default())?;
+
+        ENCRYPTION_FILE_SIZE_GAUGE
+            .with_label_values(&["file_dictionary"])
+            .set(file_bytes.len() as _);
```

Comment:
set `ENCRYPTION_FILE_NUM_GAUGE` here.

Reply:
if we don't need store encryption method as label, this could be simple